frontend/src/components/TreeView.js

```diff
@@ -37,121 +33,109 @@ const TreeView = (props) => {
-                                ? 'b'
-                                : ''
-                        }
-                        fontSize="xs"
-                        onClick={() => navigate(tree.path)}
-                    >
-                        {`${tree.type} - ${tree.name}`}
-                    </Text>
-                </Box>
+        <Stack>
+            <Box
+                bgColor="purple.200"
+                p="4px 8px"
+                mr="20%"
+                borderRadius="8px"
+                cursor="pointer"
+            >
+                <Text
+                    fontWeight={
+                        currentTicket.code === tree.code ? '800' : '600'
+                    }
+                    fontSize="xs"
+                    onClick={() => navigate(tree.path)}
+                >
+                    {`${tree.code}: ${tree.name}`}
+                </Text>
+            </Box>
 
-                {sortTickets(tree.fundingItems).map((fi) => {
-                    return (
-                        <Box key={fi.code}>
-                            <Box
-                                pos="relative"
-                                left="4"
-                                bgColor="#34cceb"
-                                cursor="pointer"
+            {sortTickets(tree.fundingItems).map((fi) => {
+                return (
+                    <Box key={fi.code} m="0 !important">
+                        <Box
+                            bgColor="blue.200"
+                            m="4px 10% 0"
+                            p="4px 8px"
+                            borderRadius="8px"
+                            cursor="pointer"
+                        >
+                            <Text
+                                fontWeight={
+                                    currentTicket.code === fi.code
+                                        ? '800'
+                                        : '600'
+                                }
+                                fontSize="xs"
+                                onClick={() => navigate(`${fi.path}`)}
                             >
-                                <Text
-                                    as={
-                                        ticketType === fi.type &&
-                                        ticketID === fi._id
-                                            ? 'b'
-                                            : ''
-                                    }
-                                    fontSize="xs"
-                                    onClick={() => navigate(`${fi.path}`)}
+                                {`${fi.code}: ${fi.name}`}
+                            </Text>
+                        </Box>
+                        {sortTickets(fi.personalPurchases).map((ppr) => {
+                            return (
+                                <Box
+                                    key={ppr.code}
+                                    bgColor="cyan.200"
```

Comment:
do we ever define custom colours? maybe its something we should start doing since we have to do it eventually. cyan.200 is just arbitrary rn

Reply:
created #98 to address this